client/src/components/Nametags/Nametag.vue

```diff
@@ -14,10 +14,10 @@ export default {
             return this.tag.startsWith("name:") ? this.tag.replace("name:", "") : this.tag;
         },
         tagStyles() {
-            const { primary, contrasting, darker } = keyedColorScheme(this.tag);
+            const { primary, darker } = keyedColorScheme(this.tag);
             return {
                 "background-color": primary,
-                color: contrasting,
+                color: "black",
```

Comment:
here maybe this should be whatever the default text colour is, used throughout the app, rather than specifically black? I'm guessing we have that set globally somewhere in the CSS, so maybe the colour can be left off completely (Not 100% sure on that)

Reply:
Ah, I see your point, if someone does a dark mode galaxy, then they'd have light text and it wouldn't work. I see your point! Never mind then.

Reply:
And dark mode is coming as far as I understood @ElectronicBlueberry ;-)